Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gem 'activerecord-import'
 gem 'activerecord-session_store'
-gem 'acts_as_list', github: 'swanandp/acts_as_list'
+gem 'acts_as_list', github: 'chrisortman/acts_as_list'
```

Comment:
I have a PR open to aal to fix a bug in position computation
https://github.com/swanandp/acts_as_list/pull/188

I'm hoping it will get merged before I'm done with this one
On Tue, Feb 23, 2016 at 6:54 AM Andrew Cates <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/sparc-request/sparc-request/pull/301#discussion_r53775623>
> :
>
> > @@ -2,7 +2,7 @@ source 'https://rubygems.org'
> >
> >  gem 'activerecord-import'
> >  gem 'activerecord-session_store'
> > -gem 'acts_as_list', github: 'swanandp/acts_as_list'
> > +gem 'acts_as_list', github: 'chrisortman/acts_as_list'
>
> Chris, why are you switching this to your version over the official
> version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sparc-request/sparc-request/pull/301/files#r53775623>.
>